cli/reference/blocks/terramate.md

```diff
@@ -4,3 +4,46 @@ description: Learn how to configure a Terramate project using the terramate bloc
 ---
 
 # The `terramate` block
+
+Use the `terramate` block to define project-wide configurations in the project's root. The `terramate` block is defined in the `terramate.tm.hcl` file and overrides any defaults by Terramate.
```

Comment:
For correctness, the `terramate` block can be defined in any file in the root directory. Usually, we place it in the `terramate.tm.hcl` but it can have any name.